doc/rados/operations/add-or-rm-osds.rst

```diff
@@ -164,6 +164,27 @@ weight).
  Note that this practice will no longer be necessary in Bobtail and
  subsequent releases.
 
+Replacing an OSD
+----------------
+
+When disk fails, or if an admnistrator wants to reprovision OSDs with a new backend, for instance, for switching from XFS to BlueStore, OSDs need to be replaced. Unlike purging an OSD, we need to keep the replaced OSD's id and CRUSH map entry intact after killing it for the replacement.
```

Comment:
i am revising the subsection of "Removing OSD" and will link to ceph(8) from there instead.